modules/bookList.js

```diff
@@ -0,0 +1,97 @@
+    listButton.style = 'color: rgba(100, 126, 255, 1);';
+    addBookButton.style = '';
+    contactButton.style = '';
+    bookListSection.style = '';
+    addBookSection.style = 'display: none;';
+    contactSection.style = 'display: none;';
+    // check the local storage if there is any database for books list
+    if (bookDB === null || bookDB.length === 0) {
+      // if booklistContainer has a child element return
+      if (bookListContainer.hasChildNodes()) {
+        return;
+    // if bookListContiner has no child put a text. it means user has no any book added at your bookList
+      } else {
+        const noteText = document.createElement('p');
+        noteText.innerText = 'Please add some books to your list';
+        noteText.className = 'note-text';
+        bookListContainer.appendChild(noteText);
+        return;
+      }
+    }
+    // if there is data, create a card for each element
+    bookDB.forEach((book) => {
+      const postElement = document.createElement("article");
+      postElement.classList.add("book-item");
+      postElement.innerHTML = `
+        <p class="book-title">"${book.title}" by ${book.author}</p>
+      `;
+      bookListContainer.appendChild(postElement);
+      // create a remove button for each book and give it the same id as the book object
+      const removeBtn = document.createElement("button");
+      removeBtn.className = "remove-btn border";
+      removeBtn.innerText = "Remove";
+      removeBtn.id = book.id;
+      postElement.appendChild(removeBtn);
+      // add event listener for each remove button to run removeBook function
+      removeBtn.addEventListener("click", this.removeBook);
+    });
+  }
+
+  addBook = () => {
+    // check bookDB if it is empty, then give it an empty array
+    const bookDB = JSON.parse(localStorage.getItem("bookData")) || [];
+    /* if inputs are not empty */
+    if (titleInput.value && authorInput.value) {
+      // create a book with input values for title and author
+      const book = new Book(titleInput.value, authorInput.value);
+      // add this book to a new array with existing array elements
+      const newBookData = [...bookDB, book];
+      // set the new array as bookDatabase
+      localStorage.setItem("bookData", JSON.stringify(newBookData));
+    }
+  }
+
+  removeBook = (event) => {
+    // check event target id. It runs for every button clicked by the user. It calls the book id that contains the clicked button
+    const id = event.target.id;
+    // call the existing database as an array
+    const bookDB = JSON.parse(localStorage.getItem("bookData"));
+    // filter the existing array to create a new array without the removed book
+    const newBookData = bookDB.filter((book) => {book.id !== id});
+    // then set it as the bookDatabase on localStorage
+    localStorage.setItem("bookData", JSON.stringify(newBookData));
+    // then reload the page to refresh the book list section
+    window.location.reload();
+  }
```

Comment:
- [ ] Kindly make sure the user can remove the unique book because now when the user tried to remove one book instead all books will be removed. Kindly fix that error.

  - ### YOUR APP BEHAVIOUR
  
   ![chrome-capture-2023-6-11 (2)](https://github.com/s2jack/awesome-books-modules/assets/87197412/393de731-9c74-4217-84fd-920fceabf550)